warrior/Framework/Utils/data_Utils.py

```diff
@@ -1446,8 +1458,15 @@ def subst_var_patterns_by_prefix(raw_value, start_pattern="${",
                                     str(raw_value[k]).replace(
                                         search_obj.group(), 'None'))
                     except SyntaxError:
-                        print_error("Syntax Error - " +
-                                    error_msg2.format(string, value))
+                        from WarriorCore.Classes.testcase_utils_class import TestcaseUtils
```

Comment:
Please put import statement at the beginning of the file

Reply:
done.